electron-builder.env

```diff
@@ -0,0 +1 @@
+BROWSER_REVISION=907428
```

Comment:
We can automate this with ENV variable. We need to get it from https://github.com/elastic/synthetics-recorder/blob/13a146375d4582a04361d057969c3ab17c6e5e42/scripts/install-pw.js#L29